Source/GameBaseFramework/Private/Input/GBFInputComponent.cpp

```diff
@@ -0,0 +1,67 @@
+#include "Input/GBFInputComponent.h"
+
+#include "Engine/GBFLocalPlayer.h"
+#include "EnhancedInputSubsystems.h"
+
+UGBFInputComponent::UGBFInputComponent( const FObjectInitializer & object_initializer )
+{
+}
+
+void UGBFInputComponent::AddInputMappings( const UGBFInputConfig * input_config, UEnhancedInputLocalPlayerSubsystem * input_system ) const
+{
+    // :TODO: Uncomment when local settings will be added to the plugin
```

Comment:
what is meant with local settings?

Reply:
local settings are per-player save data such as saving the input mappings, the resolution, etc...